package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "perseus",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:Khan/perseus.git",
+  "license": "MIT",
+  "private": true,
+  "workspaces": [
+    "packages/*",
+    "config/build"
+  ],
+  "engines": {
+    "node": ">=12"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.9",
+    "@babel/eslint-parser": "^7.16.5",
+    "@babel/eslint-plugin": "^7.17.7",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-flow": "^7.16.7",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-node-resolve": "^13.2.1",
+    "@rollup/plugin-replace": "^4.0.0",
+    "@khanacademy/eslint-config": "^0.1.0",
+    "babel-jest": "^27.4.6",
+    "babel-plugin-transform-flow-enums": "^0.0.2",
+    "eslint": "^8.13.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-disable": "^2.0.3",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-flowtype": "^8.0.3",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.1.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-monorepo": "^0.3.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-react": "^7.29.4",
+    "flow-bin": "^0.176.2",
+    "jest": "^27.2.4",
+    "jest-extended": "^2.0.0",
+    "prettier": "^2.6.2",
+    "rollup": "^2.70.2",
```

Comment:
Will we use only rollup for Perseus (instead of webpack)?  Or is this just repo infra?

Reply:
I think we may as well try rollup.  We were going to need to make a bunch of changes anyways to the bundle since we want them to be consumed by both webapp and mobile.  I added extra time to the mobile integration task to account for not using globals anymore.  I set it at 2 points, but we can bump that if you think it's going to be more work.  I think we'll be able to side-step porting the MathJax stuff we have in the webpack config now that MathJax is being injected using the "dependencies" singleton.